### ISSUE TITLE
Embed first two bytes of the git commit ID, encoded as an integer in AssemblyInformationalVersion

### ISSUE THREAD

**Issue by jobanputra** [AArnott/Nerdbank.GitVersioning#207]
The sample output should in the [.NET support documentation](https://github.com/AArnott/Nerdbank.GitVersioning/blob/master/doc/dotnet.md) has the first two bytes of the git comment ID, encoded as an integer in the AssemblyInformationVersion (pasted below). How do I configure my version.json to get a similar output?

```csharp
internal sealed partial class ThisAssembly {
    internal const string AssemblyVersion = "1.0";
    internal const string AssemblyFileVersion = "1.0.24.15136";
    internal const string AssemblyInformationalVersion = "1.0.24.15136-alpha+g9a7eb6c819";
    internal const string AssemblyName = "Microsoft.VisualStudio.Validation";
    internal const string PublicKey = @"0024000004800000940000...reallylongkey..2342394234982734928";
    internal const string PublicKeyToken = "b03f5f7f11d50a3a";
    internal const string AssemblyTitle = "Microsoft.VisualStudio.Validation";
    internal const string AssemblyConfiguration = "Debug";
    internal const string RootNamespace = "Microsoft";
}
```

**Comment by AArnott** [comment]
It's on by defualt. What are you seeing instead?

**Comment by jobanputra** [comment]
**version.json**

```json
{
  "$schema": "https://raw.githubusercontent.com/AArnott/Nerdbank.GitVersioning/master/src/NerdBank.GitVersioning/version.schema.json",
  "version": "1.2",
  "assemblyVersion": {
    "precision": "revision"
  }
}
```
**Output**
```csharp
//------------------------------------------------------------------------------
// <auto-generated>
//     This code was generated by a tool.
//     Runtime Version:4.0.30319.42000
//
//     Changes to this file may cause incorrect behavior and will be lost if
//     the code is regenerated.
// </auto-generated>
//------------------------------------------------------------------------------

[assembly: System.Reflection.AssemblyVersionAttribute("1.2.1.26893")]
[assembly: System.Reflection.AssemblyFileVersionAttribute("1.2.1.26893")]
[assembly: System.Reflection.AssemblyInformationalVersionAttribute("1.2.1+g0d690a243b")]


internal sealed partial class ThisAssembly {
    internal const string AssemblyVersion = "1.2.1.26893";
    internal const string AssemblyFileVersion = "1.2.1.26893";
    internal const string AssemblyInformationalVersion = "1.2.1+g0d690a243b";
    internal const string AssemblyName = "redacted";
    internal const string AssemblyTitle = "redacted";
    internal const string AssemblyConfiguration = "Debug";
    internal const string RootNamespace = "redacted";
    private ThisAssembly() {
    }
}
```

**Comment by AArnott** [comment]
I see. The docs are wrong/outdated. NB.GV never puts the 4th integer into the AssemblyInformationalVersion:

https://github.com/AArnott/Nerdbank.GitVersioning/blob/c518c0306a3cdcf467871acdb6fd3430b662e632/src/NerdBank.GitVersioning/VersionOracle.cs#L179-L180

Adding the first two bytes of the commit ID as an integer in the AssemblyInformationalVersion would be redundant because the git commit ID itself can be and is present in that same field.

Closed by c518c03
